client/src/templates/Challenges/classic/multifile-editor.tsx

```diff
@@ -134,13 +140,16 @@ const MultifileEditor = (props: MultifileEditorProps) => {
                 >
                   <Editor
                     canFocusOnMountRef={canFocusOnMountRef}
+                    block={block}
+                    superBlock={superBlock}
                     challengeFiles={challengeFiles}
                     containerRef={containerRef}
                     description={targetEditor === key ? description : ''}
                     editorRef={editorRef}
                     fileKey={key as FileKey}
                     initialTests={initialTests}
                     isMobileLayout={isMobileLayout}
+                    isMobileHeight={isMobileHeight}
```

Comment:
Can we import this variable directly into the editor instead?

Reply:
Thanks for your review.

I now noticed that it seems that the variable were redundantly added, so I will try to import the one directly into the editor.